src/assets/scripts/client/utilities/radioUtilities.js

```diff
@@ -400,10 +401,25 @@ export const radio_spellOut = (alphanumeric) => {
 export const radio_altitude = (altitude) => {
     const alt_s = altitude.toString();
     const s = [];
+    const transalt = AirportController.getTransAlt();
```

Comment:
Please use complete words for variables `transalt` -> `transitionAltitude`. I hate that there is a variable named `s` just above this. If you can give that a meaningful name too, that would be great but not required.